generate-export.cjs

```diff
@@ -14,6 +14,9 @@ function getFlatExportsFrom(dirPath, baseDistPath) {
     )
     .reduce((acc, file) => {
       const nameWithoutExt = path.basename(file, path.extname(file));
+      if (nameWithoutExt === "context-menu") {
```

Comment:
let's put this `context-menu` into a constant array like `IGNORED_COMPONENT_FILES` and then check for inclusion?